form_components/AppointmentForm.js

```diff
@@ -71,13 +73,21 @@ const styles = StyleSheet.create({
         marginBottom: 5,
         fontSize: 18,
         borderRadius: 6,
-        backgroundColor:'#EDEFEF'
+        backgroundColor:'#EDEFEF',
+        width: 400,
+        alignSelf: 'center'
     },
     addButton:{
         backgroundColor:'white',
         padding:10,
         width:200,
         left:100
+    },
+    formName: {
+        paddingBottom: 100,
+        textAlign: 'center',
+        fontSize: 50,
+        color: '#5e5e5d'
```

Comment:
Same comment about GlobalColors usage. If we use the same hex code in many places, please add it to GlobalColors.